propagator/opentelemetry-propagator-jaeger/tests/test_jaeger_propagator.py

```diff
@@ -230,3 +230,15 @@ def test_extract_invalid_uber_trace_id_header_to_implicit_ctx(self):
 
                 ctx = FORMAT.extract(carrier)
                 self.assertDictEqual(Context(), ctx)
+    
+    def test_non_recording_span(self):
+        """Make sure propagator does not crash when working with NonRecordingSpan"""
```

Comment:
This should describe what is being tested, and what is being tested is what is being asserted. In this case what is being asserted is the equality of `FORMAT.fields` and `inject_fields`. I understand that this test will fail if the propagator "crashes" somehow, but that is not how "not crashing" should be tested. If what you want to test is that nothing "crashes" (with that I mean that no exception is raised during the execution of the test) then the test should have this form:

```python
    def test_something(self):
        try:
            # The piece of code that needs to be tested for not crashing goes here.
            ...
        except Exception as error:
            self.fail(f"The code being tested failed with {error}")
```

Reply:
Updated this test.